synapse/tests/test_lib_cell.py

```diff
@@ -503,6 +503,10 @@ async def test_cell_getinfo(self):
             cell.COMMIT = 'mycommit'
             cell.VERSION = (1, 2, 3)
             cell.VERSTRING = '1.2.3'
+
+            await cell.addHttpsPort(0)
+            await cell.addHttpsPort(0, host='127.0.0.1')
```

Comment:
This API returns the (host, port) tuple. You could capture it here and use it to validate the expected values later.